docs/faq/is-webiny-free.md

```diff
@@ -0,0 +1,15 @@
+---
+id: is-webiny-free
+title: Is Webiny free?
+sidebar_label: Is Webiny free?
+---
+
+Yes, it is! 
+
+Depending on what you use Webiny for, your hosting prices will vary though. If you host a site that rarely changes, you will be hitting the CDN all the time, which is results in negligible expenses.
```

Comment:
`which is results` at the end of the sentence

Reply:
didn't notice that. fixed